### PR TITLE
Substitute certain lists with range objects

### LINKED ISSUE #23: Storing `range` objects in the `answers` dict instead of lists.
`range` objects are indexable and `random.choice` works on it. We're needlessly calling `list` on them which is just a waste of memory.

I'd be happy to open a PR to make the changes.

### PR DESCRIPTION
**Pull request by Qwerty-133**
Closes #23.

### CODE REVIEW

File: bestpy/best.py [hunk 1]
```diff
@@ -1,14 +1,16 @@
...
 
 class Best:
     def __getattribute__(self, name):
         answer = answers[name]
-        if callable(answer):
-            answer = answer()
 
-        if isinstance(answer, list):
+        if callable(answer):
+            return answer()
```
Comment (gustavwilliam) [inline bestpy/best.py]:
```suggestion
            answer = answer()
```
This doesn't look quite right. We're updating the previous variable to get the return value and continue the execution with it.

**Review by gustavwilliam — CHANGES_REQUESTED** [review]
Thanks for the PR! Just one minor issue. Other than that, it looks good!

File: bestpy/best.py (hunk 1 above)
Comment (Qwerty-133) [reply bestpy/best.py]: Yep, not quite sure why I made that change.

**Review by gustavwilliam — APPROVED** [review]
Looks good to me! Thanks for the PR 😄

**Pull request merged**